src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  background-color: #000;
+  margin: 0;
+}
+
+.enclosures-bordered {
+  box-sizing: border-box;
+  border: 1px solid #e75551;
+  padding: 10px;
+}
+
+.enclosures {
+  width: 350px;
+  margin: 20px 20px 40px;
+  background-color: #308dc0;
+}
+
+.enclosures-wrap {
```

Comment:
use `enclosures-wrapper` instead of `...-wrap` (better to use nouns for names, not verbs)